### PR TITLE
UG update

### LINKED ISSUE #141: [PE-D] Excessive Visuals for examples in UI
I like that your group uses a lot of visuals to guide the reader but personally I feel that the use of images is excessive when describing the expected output for each example command output as it makes your user guide unnecessary long. Perhaps you could explore only having 1 example for expected output for the command in the main body of the command, then under examples you could simply type out the expected output without the use of visuals.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: KT27Learn/ped#1

### PR DESCRIPTION
**Pull request by NayLin-H99**
Major fixes and updates to UG based on comments.

Fix #141 #157 #188

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
...
 
-3. `add recipe -n Grilled Lamb Chop with Mint Puree -i Mint 5g -i Lamb Chops 1kg -i Butter 20g -i Garlic 20g -s Season
+2. All command flags used:<br />
+Command:`add recipe -n Grilled Lamb Chop with Mint Puree -i Mint 5g -i Lamb Chops 1kg -i Butter 20g -i Garlic 20g -s Season
 the lamb chops with salt and pepper. -s Grill the lamb chops over medium high heat until cooked with butter.
 -s Blend the mint with garlic to make a puree. -s Leave the lamb chops to rest for 5min. -s Serve with mint puree.
 -d Juicy lamb chops served medium rare with a refreshing mint puree.`
-![addRecipe3.png](images/recipeCommands/addRecipe3.png)
+<br />Expected Output:<br />
+![addRecipe2.png](images/recipeCommands/addRecipe2.png)
```
Comment (alissayarmantho) [inline docs/UserGuide.md:376]: The output for steps no.2 is wrong (?) Typo right...hopefully?

Steps 1 also the space

File: docs/UserGuide.md [hunk 2]
```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
...
+    2. Ingredient(s) will be matched as long as a full keyword is in its name.<br />
+       e.g. Finding with keyword: "corn" will match with "cornflour" and "corn flakes" but not "apple core". <br />
 
-### 4.2.6 List Ingredients
+[\[Back to Table of Contents\]](#table-of-contents)
 
-List all the ingredients again after `find ingredient` Operation.
+#### 4.4.6 List Ingredient
+
+Switch to Ingredient [tab](#31-tabs) and list all ingredients.
```
Comment (alissayarmantho) [inline docs/UserGuide.md:323]: Mention sorting of ingredients smwhere? Hmm actually not sure where to put the sorting info cuz its across the commands right.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
...
 
 **Additional Information:**<br />
 - An index number is required for the Command. Refer to the indexes displayed for each [Card](#33-cards) in the
   [Side Bar](#32-side-bar).
 
-## 4.3 Recipes
-This sections covers commands related to RecipeBook management. Any command primarily interacting with recipes will
+[\[Back to Table of Contents\]](#table-of-contents)
+  
+### 4.5 Recipes
```
Comment (alissayarmantho) [inline docs/UserGuide.md:353]: Mention sorting of recipe ingredients on display somewhere..?(by name). I think recipes have no sorting.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
...
-**View Ingredient** | `view ingredient <index>`
-**Add Recipe** | `add recipe -n <name> -i <ingredient> <quantity> [-d <optional description>] [-s <steps>]...`
-**Delete Recipe** | `delete recipe <index>...`
-**Edit Recipe** | `edit recipe <index> (-<field flag> <new data>)...`
-**Find Recipe** | `find recipe <keyword>...`
-**List Recipe** | `list recipe`
-**View Recipe** | `view recipe <index>`
-**Cook Recipe** | `cook recipe <index>`
+---
+## 5. Glossary
```
Comment (alissayarmantho) [inline docs/UserGuide.md:532]: Add alphanumeric definition here.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
...
 
 **Format:**<br />
-`add ingredient -n <name> -q <quantity>[<units>] [-d <description>] -e <expiry date> [-t <tags>]`
+`add ingredient -n <name> -q <quantity> [<units>] -e <expiry date> [-d <description>] [-t <tags>]...`
 
 **Example(s):**<br />
-1. `add ingredient -n tomato -d from africa -q 500 -e 27-09-2021 -t sweet`
-    <br />Expected Output:<br />
-    ![addCommand1.png](images/ingredientCommands/addCommand1.png)
+1. All command flags used:<br />
```
Comment (alissayarmantho) [inline docs/UserGuide.md:202]: Minimal flags first then all command flags like recipe(?)

File: docs/UserGuide.md [hunk 6]
```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
...
-    <br />Expected Output:<br />
-    ![addRecipe1.png](images/recipeCommands/addRecipe1.png)
-
-2. `add recipe -n aglio olio -i pasta 200g -d grandmother aglio olio recipe -s aglioli olioli -s aglioli olioli`
-    <br />Expected Output:<br />
-    ![addRecipe2.png](images/recipeCommands/addRecipe2.png)
+1. Minimal command flags used:<br /> 
+Command: `add recipe -n pasta -i tomato 2 -i linguine 50g`
+<br />Expected Output:<br />
+![addRecipe1.png](images/recipeCommands/addRecipe1.png)
```
Comment (alissayarmantho) [inline docs/UserGuide.md:368]: Soz. I'll update the ingredients spacing when there is no quantity provided, can retake the screenshot later after it's merged?

File: docs/UserGuide.md (hunk 2 above)
Comment (rootkie) [reply docs/UserGuide.md:323]: It's under 4.4

File: docs/UserGuide.md (hunk 5 above)
Comment (NayLin-H99) [reply docs/UserGuide.md:202]: You meant you want us to swap it to match the ordering? Okay. 🐒

File: docs/UserGuide.md (hunk 3 above)
Comment (pattan9955) [reply docs/UserGuide.md:353]: Recipe ingredients are not sorted I believe

Comment (NayLin-H99) [reply docs/UserGuide.md:353]: I don't think there's any form of sorting at at all for recipes?

File: docs/UserGuide.md (hunk 5 above)
Comment (alissayarmantho) [reply docs/UserGuide.md:202]: ye

**Comment by alissayarmantho** [comment]
Also later can pull the latest master changes in.

File: docs/UserGuide.md (hunk 4 above)
Comment (NayLin-H99) [reply docs/UserGuide.md:532]: okay 🐒

File: docs/UserGuide.md (hunk 6 above)
Comment (NayLin-H99) [reply docs/UserGuide.md:368]: You meant when there are no units provided? Okay.

File: docs/UserGuide.md (hunk 3 above)
Comment (alissayarmantho) [reply docs/UserGuide.md:353]: I think recipe ingredients is sorted tho? As in the order of display compared to the order of my -i tags

Yeah recipe is not sorted

Comment (alissayarmantho) [reply docs/UserGuide.md:353]: I think recipe ingredients is sorted tho? As in the order of display compared to the order of my -i tags

Yeah recipe is not sorted

File: docs/UserGuide.md (hunk 2 above)
Comment (alissayarmantho) [reply docs/UserGuide.md:323]: Ah ok

**Review by rootkie — APPROVED** [review]
LGTM

**Pull request merged**